File: sprixel/misc/numish.js

// 
Act.types.numish = function(){
    //throw keys(this.node);
    do { switch(this.phase) {
    case 0:
        this.phase = 1;
        return this.next = new Act(this.node.integer, this);
    default: break;
    } } while (false);
    this.result = this.next.result;
    this.next = null;
    return this.invoker;
};
1;

File: vill/src/numish.cpp
// numish.cpp
#include "villCompiler.h"

Value *
numish( struct villCompiler * vill, struct graph_node * node ) {
  Value * result = NULL;
  vill -> debug_flags & DEBUG_USER && fprintf( stderr, "<numish>" );
  graph_node * integer = ast_mapentry( node, "integer" );
  assert( integer != NULL );
  function_pointer Codegen;
  Codegen = (function_pointer) integer -> data;
  assert( Codegen != NULL );
  result = Codegen( vill, integer );
  vill -> debug_flags & DEBUG_USER && fprintf( stderr, "</numish>" );
  return result;
}
